<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select_grains_dialog</class>
 <widget class="QDialog" name="select_grains_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Grains</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="GrainsTableView" name="table_view"/>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hedm_calibration_output_tab">
      <attribute name="title">
       <string>Hedm Calibration Output</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </widget>
     <widget class="QWidget" name="find_orientations_output_tab">
      <attribute name="title">
       <string>Find Orientations Output</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QWidget" name="fit_grains_output_tab">
      <attribute name="title">
       <string>Fit Grains Output</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
     <widget class="QWidget" name="file_tab">
      <attribute name="title">
       <string>File</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="file_name">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="select_file_button">
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="method_label">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="method"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GrainsTableView</class>
   <extends>QTableView</extends>
   <header>grains_table_view.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>method</tabstop>
  <tabstop>tab_widget</tabstop>
  <tabstop>table_view</tabstop>
  <tabstop>file_name</tabstop>
  <tabstop>select_file_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>select_grains_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>select_grains_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
